runtime/src/bank.rs

```diff
@@ -4520,6 +4524,54 @@ impl Bank {
         timings: &mut ExecuteTimings,
         log_messages_bytes_limit: Option<usize>,
     ) -> (Vec<TransactionCommitResult>, TransactionBalancesSet) {
+        self.do_load_execute_and_commit_transactions_with_pre_commit_callback(
+            batch,
+            max_age,
+            collect_balances,
+            recording_config,
+            timings,
+            log_messages_bytes_limit,
+            None::<fn(&mut _, &_) -> _>,
+        )
+        .unwrap()
+    }
+
+    pub fn load_execute_and_commit_transactions_with_pre_commit_callback(
```

Comment:
I feel we should just make this the have the body of `do_load_execute_and_commit_transactions_with_pre_commit_callback`, and accept an `Option`? Doesn't seem necessary to have this extra function to me

The public `None`-only version makes sense to me because of the way Rust would force calling code to specify type like `None::<fn(&mut _, &_) -> _>,`.

Reply:
Sounds good. I'm happy with the explanation